examples/demo4.yaml

```diff
@@ -103,14 +106,32 @@ gal :
     type : Sum
 
     items :
+        # 0.6 of flux is in the total disk.  Further divide this
+        # into a smooth disk and knots of star formation, modeled
+        # as point sources distributed via a random walk
+        #
+        # note the knots get the same hlr as the smooth disk, and
+        # get sheared by the same ellipticity, but the knots will
+        # themselves have additional shape noise
+
         - 
             type : Exponential
             half_light_radius : { type : Catalog , col : 5 }
             ellip : 
                 type : E1E2 
                 e1 : { type : Catalog , col : 6 }  
                 e2 : { type : Catalog , col : 7 }
-            flux : 0.6
+            flux : 0.18
```

Comment:
Well, demo11.yaml gives some examples you can follow (of defining variables and then using them in expresssions later on).  But if we add that to this demo then we have to explain about the eval capabilities here, which would be a fairly big change.  So let me change my recommendation: leave it as-is, but put a comment saying that if you don't want to hard-code such things, see demo11.yaml for an example of how to define variables and then use them for calculations within the config framework.